packages/common/src/compiler/translator.ts

```diff
@@ -248,6 +233,22 @@ export const convertTable = (
                 );
             }
 
+            const columnMetrics = Object.fromEntries(
+                Object.entries(column.meta.metrics || {}).map(
+                    ([name, metric]) => [
+                        name,
+                        convertMetric({
+                            modelName: model.name,
+                            dimensionName: dimension.name,
+                            dimensionSql: dimension.sql,
```

Comment:
Nice so we generate the metric after the dimension now and the `.sql` field is always present 💪